sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/tests/QuestionAnsweringProjectsClientTests.cs

```diff
@@ -0,0 +1,321 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using NUnit.Framework;
+using Azure.AI.Language.QuestionAnswering.Projects;
+using Azure.Core.TestFramework;
+
+namespace Azure.AI.Language.QuestionAnswering.Tests
+{
+    public class QuestionAnsweringProjectsClientTests
+    {
+        public Uri Endpoint => new("https://leithyta-sdktesting.cognitiveservices.azure.com", UriKind.Absolute);
+
+        public QuestionAnsweringProjectsClient Client => new(Endpoint, new AzureKeyCredential("5d757e79a27f40f896d1dc0c04fb3d09"));
+
+        // private String SampleProjectName = "sampleProjectName";
+
+        // private String SampleDeploymentName = "sampleDeploymentName";
```

Comment:
Nit: I'd just remove dead code.

Reply:
These variables will be referenced by the tests below once they are activated, so I decided to temporarily comment them out. I can remove them and bring them back once the generation bug is fixed if that is better.

Reply:
Naw, that's fine to leave them then. Just make sure to use `string` not `String`.